libtenzir/builtins/contexts/bloom_filter.cpp

```diff
@@ -149,7 +149,8 @@ class bloom_filter_context final : public virtual context {
       auto column = params["field"];
       if (not column) {
         return caf::make_error(ec::invalid_argument,
-                               "missing 'field' parameter");
+                               "missing 'field' parameter for bloom "
+                               "filter-based lookup");
```

Comment:
The hyphens are a bit off in this one. How about this?

> missing 'field' parameter for lookup in bloom-filter

Reply:
Applied your suggestion.